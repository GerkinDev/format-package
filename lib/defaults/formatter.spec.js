const formatter = require('./formatter');

describe('formatter', () => {
  it('prints the object with spacing', () => {
    expect(formatter({ foo: 'bar' })).toMatchSnapshot();
  });

  it('has a new line at the end', () => {
    expect(formatter({ foo: 'bar' }).endsWith('\n')).toEqual(true);
  });
});
