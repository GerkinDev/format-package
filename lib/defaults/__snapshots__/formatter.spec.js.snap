// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatter prints the object with spacing 1`] = `
"{
  \\"foo\\": \\"bar\\"
}
"
`;
